CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to
 ## Unreleased
 
 #### Added
+- Add PPID field to `execsnoop.bt`
```

Comment:
This should go in the "tools" subsection at the bottom.

Reply:
commit 1ca2529b